bin/perceval

```diff
@@ -143,6 +147,12 @@ def parse_args():
         parser.print_help()
         sys.exit(1)
 
+    if sys.argv[-1] == '--list' or sys.argv[-1] == '-l':
```

Comment:
If you run perceval with this flag and an argument you get an error:
```
$ perceval --list test
Traceback (most recent call last):
  File "/home/sduenas/devel/venv/bin/perceval", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/home/sduenas/devel/grimoirelab/perceval/bin/perceval", line 179, in <module>
    main()
  File "/home/sduenas/devel/grimoirelab/perceval/bin/perceval", line 109, in main
    raise RuntimeError("Unknown backend %s" % args.backend)
RuntimeError: Unknown backend test
```

You should move this code into an `Action` class. You have an example on this [stackoverflow question](https://stackoverflow.com/questions/45060060/argparse-argument-to-print-and-exit-disregarding-positional-arguments). Don't forget to add a `parser.exit()` call to the `__call__` method.